benchmarks/upload_scribe.py

```diff
@@ -43,19 +44,28 @@ def _upload_intern(self, messages):
     def upload(self, messages):
         if os.environ.get('SCRIBE_INTERN'):
             return self._upload_intern(messages)
-        logs = json.dumps(
-            [
-                {
-                    "category": self.category,
-                    "message": json.dumps(message),
-                    "line_escape": False,
-                }
-                for message in messages
-            ]
```

Comment:
Can we still keep named binding of serialized logs (may be useful for debugging)

Reply:
oh, you mean `'logs': json.dumps(...)`? debugging logging can be enabled in the send_to_scribe function